app/proguard-rules.pro

```diff
@@ -1,5 +1,5 @@
 -dontobfuscate
-
+-dontoptimize
```

Comment:
Does it work for you too if you don't add this, but have `minifyEnabled` (and maybe `shrinkResources`) set to `false`? (Just curious since that may be easier to control in automation)
https://github.com/mozilla-mobile/fenix/blob/master/app/build.gradle#L49-L50

Reply:
Thanks @pocmo!! looks like that may work... setting both to false no need to add `-dontoptimize` to get tests running

Reply:
Thank you for confirming! I created a PR to make this configurable via a build flag: https://github.com/mozilla-mobile/fenix/pull/16787